README.md

```diff
@@ -146,6 +146,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Actions for Logtalk and Prolog](https://github.com/logtalk-actions)
 - [GitHub Actions for Deno](https://github.com/denolib/setup-deno)
 - [GitHub Actions for Unity](https://github.com/webbertakken/unity-actions)
+- [GitHub Actions for Xamarin](https://github.com/maxim-lobanov/setup-xamarin)
```

Comment:
This seems to be a single action that sets up Xamarin as opposed to a collection of actions. It would most likely fit under the "Utility" category for now. Would you mind moving it to that section and give it the title "Setup Xamarin"?

Reply:
Fixed, thank you for your feedback. I mixed up the purpose of this section.